pkgs/by-name/ap/aphorme/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, wayland
+, libxkbcommon
+, libGL
+, testers
+, aphorme
+, autoPatchelfHook
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "aphorme";
+  version = "0.1.19";
+
+  src = fetchFromGitHub {
+    owner = "Iaphetes";
+    repo = "aphorme_launcher";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-p1ZIMMDyQWVzoeyHb3sbeV6XQwbIDoQwJU8ynI8hGUI=";
+  };
+
+  cargoHash = "sha256-aFoy5KTapx+5aIzvDwMfjxZQ6WKQtvX3h7rNX4LBeN8=";
+
+  # No tests exist
+  doCheck = false;
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  runtimeDependencies = [
+    wayland
+    libGL
+    libxkbcommon
+  ];
+  autoPatchelfIgnoreMissingDeps = [ "libgcc_s.so.1" ];
```

```suggestion

  buildInputs = [ stdenv.cc.cc.lib ];
```
seems more reliable. `autoPatchelfIgnoreMissingDeps` is barely used in the nixpkgs codebase and should probably be reserved for exceptional situations.

---

Thank you!
I did not realize that you could get the gcc dependency that way.
Has been amended to keep amount of commits down.